.github/workflows/build-report.yml

```diff
@@ -6,8 +6,8 @@ name: Generate Build Report
 on:
   workflow_dispatch:
   schedule:
-    # 1pm UTC is 6am PDT.
-    - cron: "0 13 * * *"
+    # 6AM UTC is 8AM EET, 7AM CET, 11PM PST.
```

Comment:
I think better have 3 hours before try get report, because we have some streams with timeout more than 7200. For example mixpanel has 9000 seconds timeout for basic read.

Reply:
Ok, lets push back the test start at 3AM UTC.